cloud/misc/rhvm.py

```diff
@@ -213,44 +213,44 @@
 
 EXAMPLES = '''
 # basic get info from VM
-  action: rhevm
+  action: rhvm
   args:
       name: "demo"
-      user: "{{ rhev.admin.name }}"
-      password: "{{ rhev.admin.pass }}"
-      server: "rhevm01"
+      user: "{{ rhv.admin.name }}"
+      password: "{{ rhv.admin.pass }}"
+      server: "rhvm01"
       state: "info"
 
 # basic create example from image
-  action: rhevm
+  action: rhvm

```

Comment:
@gundalow the main reason I do that is because it feels prettier in vim and makes my markup look better. This is also the reason I write all of the arguments in all of my playbooks as complex ones. Imho, it just looks a lot better with the yaml markup.looks